stitching/cli/stitch.py

```diff
@@ -295,6 +302,9 @@ def main():
     preview = args_dict.pop("preview")
     output = args_dict.pop("output")
 
```

Comment:
name this output_params and put it directly below output, we don't need the comment

Reply:
You can use `output_params = args_dict.pop("output_params", [])
`
It removes the param from the dict since stitcher dont accept it. If its not there we use an empty list

Reply:
But I dont know If output_params is not in the dict (then it would work) or None (then it would not work)

Reply:
We should just set an empty list as Default as its done with the Feature masks:

parser.add_argument(
        "--feature_masks",
        nargs="*",
        default=[],
        help="Masks for selecting where features should be detected.",
        type=str,
    )